source/tests/test_model_compression_se_atten.py

```diff
@@ -40,25 +40,27 @@ def _subprocess_run(command):
     return popen.returncode
 
 
+# 4 tests:
+# - type embedding FP64, se_atten FP64
+# - type embedding FP64, se_atten FP32
+# - type embedding FP32, se_atten FP64
+# - type embedding FP32, se_atten FP32
+tests = [
+    {"se_atten precision": "float64", "type embedding precision": "float64"},
+    {"se_atten precision": "float64", "type embedding precision": "float32"},
+    # {"se_atten precision": "float32", "type embedding precision": "float64"},
+    # {"se_atten precision": "float32", "type embedding precision": "float32"},
```

Comment:
Why these tests are disabled?

Reply:
Because the `self.recovered_switch` in the expression `two_embd = two_embd * tf.reshape(self.recovered_switch, [-1, 1])
` in se_atten.py:1163 is always a float64, if the data type of `two_embd` set to float32, there will raise an TypeError: `TypeError: Input 'y' of 'Mul' Op has type float64 that does not match type float32 of argument 'x'.`

Reply:
In this case `self.recovered_switch` should be cast to float32